### PR TITLE
Week1/solutions

### PR DESCRIPTION
**Pull request by venivedi**
Just test how the pull request  works

### CODE REVIEW

**Comment by mapshen** [comment]
You should merge your commits into member/dafang-wang instead master. Fixed it for you.

File: wk1-binary-search/problem5_leet378_kSmallestEle.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        heap1 = [(matrix[0][0], 0, 0)] #heap1
+        cnt = 0
+        while cnt < k:
+            (val, ii, jj) = heapq.heappop(heap1)
+            if jj+1 <= nn-1:
+                heapq.heappush(heap1, (matrix[ii][jj+1], ii, jj+1)) 
+            if ii+1 <= nn-1 and jj==0:
+                heapq.heappush(heap1, (matrix[ii+1][jj], ii+1, jj)) 
+            cnt += 1
+        return val
```
Comment (Vinznx) [inline wk1-binary-search/problem5_leet378_kSmallestEle.py]: Binary search might be faster than go over from left to right in each row.

**Comment by Vinznx** [comment]
May I ask why I can only see your solution of problem 5?

File: wk1-binary-search/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Week 1: Binary Search
-
+Test
```
Comment (mapshen) [inline wk1-binary-search/README.md]: remove this?

File: README.md [hunk 3]
```diff
@@ -26,3 +25,4 @@ Topics are listed as below:
 You need to update the `README.md`, which contains problems others need to solve, of your chosen topic prior to our weekly disccusion.
 
 Similarly, you will prepare the problem set on a branch (can be named like `<week-number>/problem-set`) in your **forked** repo and request to merge it into **this** repo's master branch when you are ready.
+Dafang test
```
Comment (mapshen) [inline README.md]: remove this?

File: wk1-binary-search/problem5_leet378_kSmallestEle.py (hunk 1 above)
Comment (mapshen) [inline wk1-binary-search/problem5_leet378_kSmallestEle.py]: Not sure if I've missed the other questions?

**Pull request merged**